pom.xml

```diff
@@ -22,13 +22,12 @@
         <rest-service-common-library.version>0.0.5</rest-service-common-library.version>
         <maven-compiler-plugin.version>3.8.1</maven-compiler-plugin.version>
         <maven-surefire-plugin.version>2.22.2</maven-surefire-plugin.version>
-        <ch-kafka.version>1.4.5</ch-kafka.version>
-        <structured-logging.version>1.9.11</structured-logging.version>
+        <ch-kafka.version>1.4.2</ch-kafka.version>
```

Comment:
Is this what implements kafka3? Seems strange to go back a few versions

Reply:
It does go back slightly but it's the version of ch-kafka they used in https://github.com/companieshouse/insolvency-delta-consumer/blob/main/pom.xml which is our Kafka3 example

Reply:
I don't think we need to go backwards with the version of `ch-kafka` as this is our own library. The first bit in the ticket is to make sure the version of `sping-kafka` supports `kafka3` and if you look at https://spring.io/projects/spring-kafka#overview that version is 2.8.0 and is already set in the pom at line 30 so I think its good to go. Might be worth checking with Jo though

Reply:
checking some of the other projects that use Kafka some use higher versions of ch-kafka than 1.4.2 so it should be fine to put it back to 1.4.5